docs/source/api/plots.rst

```diff
@@ -8,16 +8,10 @@ Plots are delegated to the
 `ArviZ <https://arviz-devs.github.io/arviz/index.html>`_.
 library, a general purpose library for
 "exploratory analysis of Bayesian models."
-For plots, ``pymc3.<function>`` are now aliases
-for ArviZ functions. Thus, the links below will redirect you to
-ArviZ docs:
+Refer to its documentation to use the plotting functions directly.
 
-- :func:`pymc3.traceplot <arviz:arviz.plot_trace>`
-- :func:`pymc3.plot_posterior <arviz:arviz.plot_posterior>`
-- :func:`pymc3.forestplot <arviz:arviz.plot_forest>`
-- :func:`pymc3.compareplot <arviz:arviz.plot_compare>`
-- :func:`pymc3.autocorrplot <arviz:arviz.plot_autocorr>`
-- :func:`pymc3.energyplot <arviz:arviz.plot_energy>`
-- :func:`pymc3.kdeplot <arviz:arviz.plot_kde>`
-- :func:`pymc3.densityplot <arviz:arviz.plot_density>`
-- :func:`pymc3.pairplot <arviz:arviz.plot_pair>`
+.. warning::
```

Comment:
I would not use an sphinx warning here but inside its api docs. 

```
"""Plot posterior predictive of a linear model.

    Parameters
    ----------
    trace : arviz.InferenceData or MultiTrace
        Output of pm.sample()
    eval : array_like
        Array over which to evaluate lm
    lm : function, default: linear function
        Function mapping parameters at different points
        to their respective outputs.
            input: point, sample
            output: estimated value
    samples : int, default=30
        How many posterior samples to draw.
    kwargs : mapping, optional
        Additional keyword arguments are passed to ``matplotlib.pyplot.plot()``.

    Warnings
    --------
    The `plot_posterior_predictive_glm` function will removed in a future PyMC3 release.
    """
```

something like that.

Reply:
Sounds great @OriolAbril I added it. However, the Warning did not show in my local version. 🤷 

<img width="641" alt="Screen Shot 2021-01-17 at 10 21 45 PM" src="https://user-images.githubusercontent.com/19514362/104879344-86c0d580-5912-11eb-9ee1-f7ce1bf4579e.png">